core/lib/constants/src/contracts.rs

```diff
@@ -173,3 +177,9 @@ pub const SHARED_BRIDGE_ETHER_TOKEN_ADDRESS: Address = H160([
 /// Default `ERA_CHAIN_ID`. All hyperchains start with this chain id and later on during their registration
 /// an "initial upgrade" transaction overrides it with the correct value.
 pub const DEFAULT_ERA_CHAIN_ID: u32 = 270;
+
+/// Default max number of hyperchains used for testing
+pub const MAX_NUMBER_OF_HYPERCHAINS: u32 = 100;
+
+/// Test ZK chain id
+pub const TEST_ZK_CHAIN_ID: u32 = 272;
```

Comment:
does it need to be in the common folder for all constants? I think it is used in only one test

Reply:
I thought it could be reused in the future, by I can remove both